api/taxonomies/views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import generics
+
+from api.taxonomies.serializers import TaxonomySerializer
+from api.base.views import JSONAPIBaseView
+
+import json
+
+class PlosTaxonomyMixin(JSONAPIBaseView, generics.RetrieveAPIView):
+    permission_classes = (
+    )
+    serializer_class = TaxonomySerializer
+    view_category = 'plos-taxonomies'
+
+    # overrides RetrieveAPIView
+    def get_object(self):
+        with open(self.data_file, 'r') as json_file:
```

Comment:
Let's indeed put this on the docket to revisit putting these into a model or something so we're not reading this from a file every time - will consult with others about our options! https://openscience.atlassian.net/browse/PREP-77